framework/resources/Functional/table_function/positive/plan/table_function_plan_check.json

```diff
@@ -0,0 +1,31 @@
+{
+  "testId": "table_function-functional",
```

Comment:
Please update testId to make sure it's unique (you have the same for both data and plan tests).